src/material/slider/slider.spec.ts

```diff
@@ -1686,7 +1686,7 @@ function setValueByClick(
   input.focus();
   dispatchPointerEvent(inputElement, 'pointerup', x, y);
   dispatchEvent(input._hostElement, new Event('change'));
-  tick();
+  tick(10);
```

Comment:
can this just be `tick()`? now that we removed the 10

Reply:
Yep, fixed